### PR TITLE
Update some licenses

### PR DESCRIPTION
**Pull request by reinerh**
Hi @cxong,

I'm a Debian maintainer and [intend to package](https://bugs.debian.org/1040944) cdogs-sdl for Debian.
I'm currently reviewing licenses of data files and found some inconsistencies/updates (many files documented as CC-BY-3.0 are (meanwhile?) CC-BY-4.0).

Also it looks like flamer_pickup.txt actually describes the file flamer_switch.ogg (as no flamer_pickup.ogg exists, and flamer_switch.ogg misses a .txt file; and from listening to the files I think the rename is correct).

explosion_small.txt says that it is derived from Explosion_001.mp3, which is (now?) licensed as CC-BY-NC-4.0. Is it possible to replace this file, as this license is too restrictive?

sounds/hits/gas/ is missing license information. Can you please add one?

sounds/package_r.txt is lacking some information. Did you create this sound file?

I noticed src/tests/*.c have no copyright headers. I assume they are licensed as BSD-2-Clause like your other code (as they have been commited by you)?

The following files are no longer available at their original location:
http://freesound.org/people/superfreq/sounds/191583/
http://freesound.org/people/MentalSanityOff/sounds/153557/
https://www.freesound.org/people/thecluegeek/sounds/141364/
(So I'll just use the license info that is in the file)

Thanks for your response. I might have more questions and updates in the following days/weeks.

### CODE REVIEW

**Comment by reinerh** [comment]
I just found out that src/cdogs/cwolfmap/mame/fmopl.* is (re-)licensed under GPL-2+.
https://github.com/mamedev/mame/commit/ec5b9deadee9a2f95e0691f7246a8f4663ba3ba9
Though I guess at the time the file was imported by the wolf3d project (can you please clarify where cwolfmap/ is coming from) it was still under the old MAME license.

File: sounds/ammo_low.txt [hunk 1]
```diff
@@ -1,3 +1,3 @@
 Derived alert.wav by Corsica_S
 http://freesound.org/people/Corsica_S/sounds/156453/
-http://creativecommons.org/licenses/by/3.0/
+http://creativecommons.org/licenses/by/4.0/
```
Comment (cxong) [inline sounds/ammo_low.txt:3]: This must be freesound updating their licenses, as I usually copy the CC link from the page when creating these files

**Review by cxong — APPROVED** [review]

**Comment by cxong** [comment]
The mame sources were taken from ecwolf which still had the mame license. I can try updating it to the latest version from mame but if there are code changes, I'll need to test first

File: sounds/ammo_low.txt (hunk 1 above)
Comment (reinerh) [reply sounds/ammo_low.txt:3]: I don't think they update this automatically, as many sounds still use 3.0.  
I think the creators manually switched to 4.0 (maybe freesound is asking them to update?).

**Comment by reinerh** [comment]
Thanks for the updates. No need to update the mame sources, it's fine like it is. :-)

**Pull request merged**